libs/extensions/angular/image-banner/src/image-banner.component.stories.ts

```diff
@@ -106,3 +106,35 @@ export const NoDismiss: Story = {
     template: `<kirby-x-image-banner ${argsToTemplate(args)}></kirby-x-image-banner>`,
   }),
 };
+
+/**
+ * The component adapts to the containers width, and thus should be plug and
+ * play with the kirby css grid and slider utilities.
+ * It switches between a narrow and wide view.
+ */
+export const UsageInGrid: Story = {
+  args: {
+    title: 'Banners that adapt in grid',
+    bodyText: 'This is the body text.',
+    actionButtonText: 'Read more',
+    imagePath: 'assets/images/leaves.jpg',
+    dismissClick: undefined,
```

Comment:
@henrikvoetmand suggested to remove this so the dismiss button shows, and also get the externalLink button shown to get that use-case validated when we screenshot test.

```suggestion
    externalLink: 'http://www.kirby.design',
```